### PR TITLE
Spaced practice tooltip and instructions

### PR DESCRIPTION
**Pull request by jivewise**
Spaced practice tooltip:
screen shot 2016-05-26 at 9 56 29 pm
Incorrect review instructions:
screen shot 2016-05-26 at 11 54 36 pm
Content from:
screen shot 2016-05-26 at 9 57 04 pm

https://www.pivotaltracker.com/story/show/118103083

### CODE REVIEW

**Comment by philschatz** [comment]
Should the `Content from:` be a clickable link? Also, it looks like Travis isn't happy

**Comment by jivewise** [comment]
Yes, I'm looking into fixing the tests now.  Do you know how I can make a link to the specific section?  It seems like we only get the section and chapter numbers, along with the title.  No url to link to though...

**Comment by philschatz** [comment]
Do you get the `page_chapter_section` array or is BE just providing a string with `1.1 -` hardcoded in it?

**Comment by jivewise** [comment]
I believe I get something like: `chapter_section: "1.3"`

**Comment by philschatz** [comment]
hmm, I thought `chapter_section` was at least an array but maybe to make it clickable all `chapter_section` should become an object so we can include the page UUID for the ref book, or have a sibling to `chapter_section` (like `chapter_section_uuid` that contains the book UUID for the ref book).

That'd probably be a bugfix PR because it would require changing BE stuff. But I think this is ok for now.

/cc @openstax/tutor-be

File: src/components/exercise/group.cjsx [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 ExerciseIdentifierLink = require '../exercise-identifier-link'
+
 DEFAULT_GROUP =
   show: false
+REVIEW_GROUP =
+  show: true
+  label: 'Review'
+  tooltip: '''Did you know? Research shows you can strengthen your memory —
+    and spend less time studying — if you revisit material over multiple study sessions.
+    OpenStax Concept Coach will include review questions from prior sections to give your
```
Comment (philschatz) [inline src/components/exercise/group.cjsx]: I think this changes depending on if the component is used in CC or in Tutor (see mockups), no?

Comment (jivewise) [inline src/components/exercise/group.cjsx]: Ah, right, I'll add in the project conditional.

**Comment by jivewise** [comment]
Closing this in favor of #73.

**Pull request closed without merging**